### ISSUE TITLE
Remove hour parameter from time field for daily products

### ISSUE THREAD

**Issue by clebreton** [dcs4cop/xcube-viewer#280]
For cubes with time averaged data (here daily) it does not make sense to keep the hour parameter visible, as seen below.
the hour is then often listed as 00:00 UTC (and as pictured here shows as 02:00 for CEST) which does not make sense.
![image](https://user-images.githubusercontent.com/3809213/232773839-5a5783b2-9552-4cb0-9af1-212ca968df02.png)

**Comment by forman** [comment]
Right, good point! But we currently have no clue whether the data in a time span has been aggregated or if it comprises an overlay of observations or it is a specific observation. We'd need to agree on some metadata field and then add it to the cubes or the xcube server.

**Comment by clebreton** [comment]
For the water-services cubes at least we do have this information: either in the cube names (e.g. ws-s2-l3-llur for a daily aggregated cubes, ws-s2-l2-hamburg for a L2C cube) or the metadata parameter processing_level which either has L3 or L2C

**Comment by forman** [comment]
Filename processing is not reliable, but the metadata parameter `processing_level` is fantastic! Will use that.
